dlp/snippets/src/test/java/dlp/snippets/JobsTests.java

```diff
@@ -125,4 +125,22 @@ public void testDeleteJobs() throws Exception {
     String output = bout.toString();
     assertThat(output).contains("Job deleted successfully.");
   }
+
+  @Test
+  public void testCreateDatastoreJobWithScc() throws Exception {
+    InspectDatastoreSendToScc.inspectDatastoreSendToScc(
+        PROJECT_ID, DATASTORE_NAMESPACE, DATASTORE_KIND);
+
+    String output = bout.toString();
+    assertThat(output).contains("Job created successfully");
+    String dlpJobName = output.split("Job created successfully: ")[1].split("\n")[0];
+
+    // Delete the created Dlp Job
+    DeleteDlpJobRequest deleteDlpJobRequest =
+        DeleteDlpJobRequest.newBuilder().setName(dlpJobName).build();
+
+    try (DlpServiceClient client = DlpServiceClient.create()) {
+      client.deleteDlpJob(deleteDlpJobRequest);
+    }
```

Comment:
If you plan to use `DlpServiceClient` to delete jobs in multiple test cases, consider moving the client initialization to @BeforeAll method and reuse the client across the test cases.